src/rdkafka_partition.c

```diff
@@ -1942,7 +1942,14 @@ static void rd_kafka_toppar_op (rd_kafka_toppar_t *rktp,
 	rd_kafka_toppar_op0(rktp, rko, replyq);
 }
 
-
+static int rd_kafka_toppar_fwd_app (rd_kafka_toppar_t *rktp)
+{
+        int result = 0;
+        rd_kafka_toppar_lock(rktp);
+        result = rktp->rktp_fetchq->rkq_flags & RD_KAFKA_Q_F_FWD_APP;
```

Comment:
normalise the return value and document it.
E.g.:
`result = !!(rktp->rktp_fetchq-> ... )` or ternary if .. ? 1 : 0;